Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -52,6 +52,7 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using static Orts.Simulation.RollingStocks.MSTSLocomotive;
```

Comment:
Not comfortable with this reference from a parent class back to a child class.
Instead you could delete it and add a property in the child class as I suggested elsewhere.

Reply:
Hopefully this has been corrected with changes that I have added.